src/renderer/util/index.ts

```diff
@@ -41,26 +62,66 @@ export async function determineFilesFromNestedPaths(
 ): Promise<string[]> {
   const filePaths = await Promise.all(
     paths.flatMap(async (fullPath) => {
-      const stats = await fsPromises.stat(fullPath);
-      if (!stats.isDirectory()) {
-        return [fullPath];
-      }
-      const canRead = await canUserRead(fullPath);
-      if (!canRead) {
-        throw new Error(`User does not have permission to read ${fullPath}`);
-      }
-      const pathsUnderFolder = await fsPromises.readdir(fullPath, {
-        withFileTypes: true,
-      });
-      return pathsUnderFolder
-        .filter((f) => f.isFile())
-        .map((f) => resolve(fullPath, f.name));
+      return determineFilesFromNestedPath(fullPath);
     })
   );
 
   return uniq(flatten(filePaths));
 }
 
+/**
+ * Use a given filepath's extension to determine if it is a "multifile".
+ * @param filePath Path to the file
+ */
+export function determineIsMultifile(filePath: string): boolean {
+  ///////////////////////////////////////////////////////////////////
+  // TODO - Remove this once multifile support is feature-complete //
+  //         Flip the boolean to "true" for testing                //
+  ///////////////////////////////////////////////////////////////////
+  const USE_MULTIFILE_ASSUMPTION = process.env.USE_MULTIFILE_ASSUMPTION !== undefined
```

Comment:
`process.env.USE_MULTIFILE_ASSUMPTION !== undefined && process.env.USE_MULTIFILE_ASSUMPTION` would be a little more concise, arguably less readable (and very Matty-esque)

Reply:
I think I'll leave it. Hope is I'll be removing this whole chunk soon-ish anyways, right?